### PR TITLE
complete auto loss alignment pipeline

### PR DESCRIPTION
**Pull request by L1aoXingyu**
运行 `bash loss_compare.sh` 既可以自动开始运行 loss 对齐，会在 `loss_compare` 文件夹下面生成结果，可以周期性运行一次，确保所有的模型仍然处在对齐的状态

image

后续增加每个模型，如果有对齐环节，需要在对齐之后加入到这个分支，放在 `projects/` 下
1. 需要提前跑好 torch 的结果，将 1000 iters 的 loss 生成的 txt 文件放到对应的项目下面，比如 `projects/bert_loss_compare/torch_loss.txt`；
2. 将用到的数据集和 torch 权重放到 loss 对齐的服务器上；
3. 将该模型对齐的所有代码放到 `projects/xxx` 下，不和 `libai` 里面的代码产生冲突；
4. 在 `loss_compare.sh` 中增加对应的模型训练以及 loss 曲线绘制流程；

image

### CODE REVIEW

File: projects/bert_loss_compare/train_net.py [hunk 1]
```diff
@@ -23,27 +23,37 @@
...
 from libai.trainer import DefaultTrainer, default_setup
 from libai.utils.checkpoint import Checkpointer
 
 
 class Trainer(DefaultTrainer):
+    
+    # Remove checkpointer
+    def build_hooks(self):
+        ret = [hook for hook in super().build_hooks() if not isinstance(hook, hooks.PeriodicCheckpointer)]
+        return ret
```
Comment (CPFLAME) [inline projects/bert_loss_compare/train_net.py:36]: 这里是不是只要写一个`[hooks.PriodicCheckpointer]` 就可以了?

Comment (L1aoXingyu) [reply projects/bert_loss_compare/train_net.py:36]: 这是是为了去掉 checkpointer，不要保存模型

**Review by rentainhe — APPROVED** [review]

File: projects/bert_loss_compare/train_net.py (hunk 1 above)
Comment (CPFLAME) [reply projects/bert_loss_compare/train_net.py:36]: 奥对 我看劈叉了.

**Review by CPFLAME — APPROVED** [review]
